### DISCUSSION TITLE
希望能够自由更换节点，目前很多网站不能访问

### DISCUSSION THREAD

**Discussion by tesla-vip** [getlantern/lantern-forum-cn#719] [Q&A]
希望能够自由更换节点，目前很多网站不能访问，比如切换到美国节点或者日本节点，希望这个功能加到下一版本

**Comment by MaxMKMax** [comment]
@tesla-vip 嗯嗯. 好的. 替你您反馈了. 我们在弄着, 因为地理服务器的问题所以延迟了.

**Reply by lsq88334753** [reply]
@Mike777Khoo 有个大概得时间点么，上一次反馈这个问题已经半年了，先简单一点干掉香港节点可行？连上香港节点什么都做不了。

**Reply by lsq88334753** [reply]
@Mike777Khoo 有个大概得时间点么，上一次反馈这个问题已经半年了，先简单一点干掉香港节点可行？连上香港节点什么都做不了。

**Reply by MaxMKMax** [reply]
@lsq88334753 您好. 我也反馈过很多次, 但有些限制所以导致延迟了. 我再帮你们反馈. 非常抱歉让你们久等了.
